### ISSUE TITLE
Shouldn't "dist" directory be git ignored ?

### ISSUE THREAD

**Issue by GMartigny** [anymod/vue-components#1]
Hi,

The "dist" directory is build from the "src" one. In my opinion, it shouldn't be pushed into VCS.  
Otherwise, any PR editing or adding a component need to run the build script individually. Which can lead to errors if not correctly done.

What's you thoughts on that ?

**Comment by tyrw** [comment]
From a workflow perspective I agree.  However, we set it up to use auto-deploy hooks on Heroku, which pulls from GitHub, so the dist files need to be there for that.

**Comment by GMartigny** [comment]
You could (don't know if you want though) use a Continuous Integration system to build it for you.
Any way, if someone do a PR, should they run the "build" task ?

**Comment by tyrw** [comment]
Yes, please run the "build" task, that way we can merge and deploy 😃
